home/WEB-INF/jsp/infrastructure/home/home-about-box.jsp

```diff
@@ -1,6 +1,14 @@
 <%@ include file="/WEB-INF/jsp-include/tag-import.jsp" %>
+<%@ page import="org.zfin.framework.featureflag.FeatureFlagEnum" %>
 
 <z:section title="About ZFIN">
+    <!-- Example Feature Flag Use -->
+    <c:if test="${zfn:isFlagEnabled(FeatureFlagEnum.CURATOR_JOB_POSTING)}">
+        <p class="lead">
+            <a href="https://zfin.atlassian.net/wiki/spaces/jobs/pages/2996503058/ZFIN+Jobs">
```

Comment:
this is the line that may need change every time we wanted to display this announcement as we may point to different wiki pages. I am not absolutely sure. If that would always be the same it would just work nicely.

Reply:
Yvonne confirmed that the wiki page for the curator job will always be the same wik page. So no concern then any more.